tests/test_parametric_workflow.py

```diff
@@ -1,319 +1,93 @@
-# Delete the old parametric study
-
-study_1.delete()
-
-parametricStudies_exp = 1
-parametricStudies_test = len(ParametricStudy._all_studies.keys())
-assert parametricStudies_test == parametricStudies_exp
-
-#########################################################################
-# Save the parametric project and close Fluent
-
-project_filepath = str(Path(pyfluent.EXAMPLES_PATH) / "static_mixer_study.flprj")
-
-session.solver.tui.file.parametric_project.save_as(project_filepath)
-
-assert (Path(pyfluent.EXAMPLES_PATH) / "static_mixer_study.flprj").exists() == True
-
-session.exit()
-
-#########################################################################
-# Launch Fluent again and read the previously saved project
-
-session = pyfluent.launch_fluent(precision="double", processor_count=2)
-project_filepath_read = str(Path(pyfluent.EXAMPLES_PATH) / "static_mixer_study.flprj")
-
-proj = ParametricProject(
-    session.solver.root.file.parametric_project,
-    session.solver.root.parametric_studies,
-    project_filepath_read,
-)
-
-#########################################################################
-# Save the current project
-
-proj.save()
-
-#########################################################################
-# Save the current project to a different file name
-
-project_filepath_save_as = str(
-    Path(pyfluent.EXAMPLES_PATH) / "static_mixer_study_save_as.flprj"
-)
-proj.save_as(project_filepath=project_filepath_save_as)
-
-assert (
-    Path(pyfluent.EXAMPLES_PATH) / "static_mixer_study_save_as.flprj"
-).exists() == True
-
-#########################################################################
-# Export the current project
-
-project_filepath_export = str(
-    Path(pyfluent.EXAMPLES_PATH) / "static_mixer_study_export.flprj"
-)
-proj.export(project_filepath=project_filepath_export)
-
-assert (
-    Path(pyfluent.EXAMPLES_PATH) / "static_mixer_study_export.flprj"
-).exists() == True
-
-#########################################################################
-# Archive the current project
-
-proj.archive()
-
-assert (Path(pyfluent.EXAMPLES_PATH) / "static_mixer_study.flprz").exists() == True
-
-#########################################################################
-# Close Fluent
-
-session.exit()
+from ansys.fluent.parametric import ParametricStudy
+
+
+@pytest.mark.integration
+@pytest.mark.setup
+def test_param_static_mixer(load_static_mixer_parameter):
+    session = load_static_mixer_parameter
+    session.solver.tui.solve.set.number_of_iterations("100")
+    session.solver.tui.solve.monitors.residual.criterion_type("0")
+    case_path = str(Path(pyfluent.EXAMPLES_PATH) / "save_1.cas.h5")
+    session.solver.tui.file.write_case(case_path)
+    assert (Path(pyfluent.EXAMPLES_PATH) / "save_1.cas.h5").exists() == True
+    study_1 = ParametricStudy(session.solver.root.parametric_studies).initialize()
+    parametric_studies_exp = 1
+    parametric_studies_test = len(ParametricStudy._all_studies.keys())
```

Comment:
```suggestion
    parametric_studies_test = len(study_1.get_all_studies().keys())
```
This is a class method; with my changes, it becomes an instance method, as necessary. This call works for both.